packages/eslint-plugin-orbit-components/docs/rules/unique-id.md

```diff
@@ -16,11 +16,11 @@ const App = () => (
 ```
 
 ```jsx
-import React, { useMemo } from "react";
+import React from "react";
 import randomID from "../utils/randomID";
 
-const firstID = useMemo(() => randomID("firstID"), []);
-const secondID = useMemo(() => randomID("secondID"), []);
```

Comment:
How can we test this? In this situation, this `useMemo` is actually preventing a new ID from being generated everytime. By removing `useMemo`, a new ID will be generated every time. I don't know the consequences of it, because I don't know how to test this plugin…

Reply:
Actually, this is something that we might want to remove entirely because we have a hook `useRandomId` and `useRandomIdSeed` that is working with SSR. 

But that would be breaking and outside of the scope of this PR 

I do not see a problem with having a new id on rerender, it won't affect component behavior and it does not have any performance cost, if needed to keep it the same, I guess we can use the same approach with useRef.